src/clj/chronograph/db/invite.clj

```diff
@@ -0,0 +1,35 @@
+(ns chronograph.db.invite
+  (:require [chronograph.db.core :as db]
+            [next.jdbc.sql :as sql]))
+
+(defn create!
+  ([organization-id email]
+   (create! db/datasource organization-id email))
+  ([tx organization-id email]
+   (sql/insert! tx
+                :invites
+                {:organization-id organization-id
+                 :email email}
+                db/sql-opts)))
+
+(defn find-by-id
+  [tx invite-id]
+  (first (sql/find-by-keys tx
```

Comment:
I realized that `sql/get-by-id` simplifies the code. Use at your discretion.

Reply:
We decided not to change this.